src/manifolds/ProbabilitySimplex.jl

```diff
@@ -0,0 +1,369 @@
+@doc raw"""
+    ProbabilitySimplex{n} <: AbstractEmbeddedManifold{ℝ,DefaultEmbeddingType}
+
+The (relative interior of) the probability simplex is the set
+````math
+Δ^n := \biggl\{ p ∈ ℝ^{n+1}\ \big|\ p_i > 0 \text{ for all } i=1,…,n+1,
+\text{ and } ⟨\mathbb{1},p⟩ = \sum_{i=1}^{n+1} p_i = 1\biggr\},
+````
+where $\mathbb{1}=(1,…,1)^{\mathrm{T}}∈ ℝ^{n+1}$ denotes the vector containing only ones.
+
+This set is also called the unit simplex or standard simplex.
+
+The tangent space is given by
+````math
+T_pΔ^n = \biggl\{ X ∈ ℝ^{n+1}\ \big|\ ⟨\mathbb{1},X⟩ = \sum_{i=1}^{n+1} X_i = 0 \biggr\}
+````
+
+We employ this set and its tangent spaces with the [`FisherRaoMetric`](@ref) exploiting
+the isometry to the $n$-sphere of radius 2, i.e. a scaled version of the [`Sphere`](@ref).
```

Comment:
These are technically 2 different approaches that happily coincide here. So we can say something like "The manifold is implemented assuming the Fisher-Rao metric for the multinomial distribution, which is equivalent to the induced metric from isometrically embedding the probability simplex in the $n$-sphere of radius 2."